build/azure-pipelines/linux/product-build-linux.yml

```diff
@@ -169,6 +169,13 @@ steps:
       displayName: Install dependencies
       condition: and(succeeded(), ne(variables.NODE_MODULES_RESTORED, 'true'))
 
+  # Ref https://github.com/microsoft/vscode/issues/189019
+  - ${{ if eq(parameters.VSCODE_QUALITY, 'oss') }}:
```

Comment:
You can ident the script task below into the above `else` condition and remove the current line in favor of it.